components/load_from_files/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM --platform=linux/amd64 python:3.8-slim
```

Comment:
Could you add a readme that detailes what this component does, what it supports and how to use it ?

Reply:
Sure, Ill add Readme along with these details